postgres-to-snowflake-with-cdc/docker-compose.yml

```diff
@@ -0,0 +1,44 @@
+---
+services:
+  ngrok:
+    image: ngrok/ngrok:latest
+    container_name: ngrok
+    # Sign up for an ngrok account at https://dashboard.ngrok.com/signup
+    # Get your auth-token from https://dashboard.ngrok.com/get-started/your-authtoken
+    # and either put it directly in the file here, or write it to a .env file in 
+    # the same folder as this Docker Compose file in the form
+    # NGROK_AUTH_TOKEN=your_token_value
+    command: tcp postgres:5432 --log stdout --authtoken $NGROK_AUTH_TOKEN
+    ports:
+      - 4040:4040 # Web dashboard for ngrok
+
+  postgres:
+    container_name: postgres
+    image: quay.io/debezium/example-postgres:2.3
```

Comment:
```suggestion
    image: quay.io/debezium/example-postgres:3.0
```

Reply:
any compelling reason to change this? I'd rather ship an older version that I have tested.

Reply:
Yeah, it's an arbitrary old version which may make this example look outdated. If you want to stay on 2.x, I'd use the latest there (2.5 as of now), though I don't expect any issues with just moving to 3.0 (those example images don't change much between versions; arguably, they should not follow Debezium versions in the first place).